afex_integration/views/account_view.xml

```diff
@@ -20,9 +20,13 @@
     <field name="inherit_id" ref="account.view_account_journal_form"/>
     <field name="arch" type="xml">
         <field name="currency_id" position="after">
+            <field name="aud_currency_id" invisible="1"/>
```

Comment:
just have the boolean on "can_direct_debit" which, for the moment, is if the currency is aud...